tests/Integration/Behaviour/Features/Scenario/Product/Combination/combinations_listing.feature

```diff
@@ -181,31 +181,6 @@ Feature: Generate attribute combinations for product in Back Office (BO)
       | product1SBlack | Size - S, Color - Black | CCC       | [Size:S,Color:Black] | 10              | 50       | false      |
       | product1SBlue  | Size - S, Color - Blue  | BBB       | [Size:S,Color:Blue]  | 1               | 100      | false      |
       | product1SWhite | Size - S, Color - White | AAA       | [Size:S,Color:White] | -1              | 10       | true       |
-    When I search product "product1" combinations list by following search criteria:
```

Comment:
Why do you remove this test? We still need to order by quantity even if the combination field was moved Besides I thought you handled this in the query builder so it should still work no?